pkg/api/registered/registered.go

```diff
@@ -72,6 +74,11 @@ func init() {
 	RegisteredGroupVersions = supportedVersions
 }
 
+// ValidateGroupVersion declares a groupVersion as valid.
+func ValidateGroupVersion(groupVersion unversioned.GroupVersion) {
```

Comment:
Sure. I also realized "validate" has multiple meanings.

This appending to validGroupVersions, which is used to check if the groupVersion specified in KUBE_API_VERSIONS are valid.

Without this function, if a thirdparty wants to add their API group, they will need to modify this file. This function decouples the API machinery from API.